### PR TITLE
added positions in the treatment and diagnosis resource

### PR DESCRIPTION
**Pull request by erwinspeybroeck**

### CODE REVIEW

File: types/icarPositionType.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "description": "The possible positions for treatment or diagnosis",
+
+  "type": "object",
+
+  "properties": {
+    
+    "position": {
```
Comment (ahokkonen) [inline types/icarPositionType.json:8]: Maybe this should be extracted to the separate enum file and values adjusted according to the naming convensions (aka LEGS_FRONT_LEFT -> FrontLeftLeg)?

**Review by ahokkonen — COMMENTED** [review]
Left some comments about position enumeration

File: types/icarPositionType.json (hunk 1 above)
Comment (cookeac) [reply types/icarPositionType.json:8]: I agree, this would make it easier to extend in future.

Comment (cookeac) [reply types/icarPositionType.json:8]: Agreed at the meeting today that I will make this change and then make a PR with it. So I will merge this PR and then make the change.

**Pull request merged**